src/AbstractPPL.jl

```diff
@@ -5,7 +5,7 @@ export VarName, getsym, getindexing, getlens, inspace, subsumes, varname, vsym,
 
 
 # Abstract model functions
-export AbstractProbabilisticProgram, condition, decondition, logdensity
+export AbstractProbabilisticProgram, condition, decondition, logdensityof, densityof
```

Comment:
Is densityof defined somewhere?

Reply:
Defaulted in DensityInterface, yes. https://github.com/JuliaMath/DensityInterface.jl/blob/ccade11dec6920cbc19852650e565d10254f1285/src/interface.jl#L156

Reply:
Ok